### PR TITLE
use semver in the version check

### PR DESCRIPTION
**Pull request by tomjn**
Fixes 3.5 saying that 3.4.1 is newer

### CODE REVIEW

File: php/notices.php [hunk 1]
```diff
@@ -13,12 +13,81 @@
...
+		else {
+			return -1;
+		}
+	}
+	if (v1parts.length != v2parts.length) {
+		return -1;
+	}
+	return 0;
+}
 fetch('https://raw.githubusercontent.com/Varying-Vagrant-Vagrants/VVV/master/version').then(function(response) {
```
Comment (msaggiorato) [inline php/notices.php:84]: Don't we have a way of knowing if we're on instead? Or is it not worth it?

Comment (tomjn) [reply php/notices.php:84]: Not sure I follow, can you rephrase? `master`'s version is the current release, so if we're the same or ahead then no updates are needed

Comment (msaggiorato) [reply php/notices.php:84]: Yeah, you're right, if we're on develop locally, there's no need to check for a version update for that, as we don't typically change the version on develop.

I meant that if we were on develop, check against develop branch so that you have an updated version of develop. But we don't do such thing as develop versions.

TLDR; nothing said here
